### PR TITLE
Add mac install instructions to readme, update test runner to be mac compatible

### PR DESCRIPTION
**Pull request by rushby**
Also added the passed tests count to the test result output.

### CODE REVIEW

File: e2e-tests/test_runner.sh [hunk 1]
```diff
@@ -107,14 +121,15 @@ echo -e "┌──────────────────────
...
+printf  "│ ${TXT_BIBLK} Failed tests:          ${TXT_BIRED} %3d ${TXT_NORML}                                          │\n" "${TOT_FAIL_TESTS}"
+
 if [ ${TOT_FAIL_TESTS} -ne 0 ]; then
     echo -e "├────────────────────────────────────────────────────────────────────────┤"
-    printf  "│ ${TXT_BIBLK} Failed tests:          ${TXT_BIRED} %3d ${TXT_NORML}                                          │\n" "${TOT_FAIL_TESTS}"
     for failed_test in "${FAILED_TESTS[@]}"; do
         printf "│     - %-64s │\n" "${failed_test}"
     done
-    echo -e "└────────────────────────────────────────────────────────────────────────┘"
-    exit 1
```
Comment (la10736) [inline e2e-tests/test_runner.sh:117]: I guess that there's a little undesirable side effect by remove these lines: if some tests fails the script not fail.
That could be an issue both for the developer that cannot leverage on script output to understand in the some test failed and for the CI.

Ok, removing the footer duplication could be nice but we should define an exit value variable to recover the right behavior.

**Review by la10736 — CHANGES_REQUESTED** [review]
Just recover the old behavior: if some e2e test fails `tests_runner.sh` should exit with error.

File: e2e-tests/test_runner.sh (hunk 1 above)
Comment (rushby) [reply e2e-tests/test_runner.sh:117]: Thanks both, i've added the exit status back in.

Comment (la10736) [reply e2e-tests/test_runner.sh:117]: @drgora : @JackPiri already open a PR to fix it (I've raised a red flag yesterday when I saw that Daniele's PR fixed some format issues on my last PR.

Comment (drgora) [reply e2e-tests/test_runner.sh:117]: Great for the other PR, thank you.

**Review by drgora — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**